Instead of returning nil, return string for unknown types

sroebert opened the pull request:
I am not really sure if this is the way to go, but this pull request changes the way unknown values are returned. Instead of returning `.null`, now the string is returned directly in a `Node`. This allows to work with custom PostgreSQL types as strings.

Commit:
Now falling back to string in case of unknown type.

sroebert commented:
If this enhancement is approved as correct behaviour, I will add some unit tests for it.

natebird commented:
So you could use this for UUID, JSONB, and other unsupported types?

sroebert commented:
Exactly, I am using it in one of my projects for UUIDs. Considering Fluent doesn't support it yet, this will do for now. Also I think it is nicer to at least return the value, instead of just null, considering there is actually a value.

natebird commented:
I think this is a good idea. It allows support for types that aren't natively supported (yet). We can get @tannernelson to weigh in as well.

tanner0101 commented on Sources/PostgreSQL/Node+Oid.swift:
```diff
@@ -21,7 +21,8 @@ public enum OID: Oid {
 	init(oid: Oid, value: String) {
 		guard let type = OID(rawValue: oid) else {
-			self = .null
+            // Always fallback to string, allowing to use with custom types as strings
```
Comment formatting appears a bit off here.

sroebert replied:
Fixed

tanner0101 reviewed (changes requested):
Could we add a test for this? Otherwise it's possible there could be regression.

Commit:
Changed spaces to tabs to match file

Commit:
Added unit test; Reverted back .string for unknown type as it does not make sense.

sroebert commented:
I added a unit test and reverted the `.string` for the `unknown` type. For the `unknown` type it does not really seem to make any sense. I could not even get any data in for that type.

tanner0101 commented:
LGTM, go ahead and tag
